java/data-management/strings-i/determining-string-equality.md

```diff
@@ -53,11 +55,13 @@ if(string1 == string2){
 return false;
 ```
 
+**NOTE:** If we do not use the keyword `new`, then the previous code returns `true`, this is because the variable `string2` points to the same reference as the variable `string1` (`"Enki"` is already in the String pool).
```

Comment:
```suggestion
**NOTE:** If we do not use the keyword `new`, the previous code would return `true`. This is because the variable `string2` would end up pointing to the same reference as the variable `string1` (since the string `"Enki"` would already be in the String pool by the time `string2` is created).
```

Reply:
Thanks!